.github/workflows/CI-unittests.yml

```diff
@@ -25,14 +25,18 @@ jobs:
           - os: ubuntu-latest
             python-version: "3.11"
```

Comment:
also 3.12 here?

Reply:
Not here - we are still testing as part of the matrix for 3.11 (plus 3.12, 3.13). I think there is no reason to increase the minimum python version required.